src/site/components/fullwidth_banner_alerts.drupal.liquid

```diff
@@ -47,14 +47,14 @@
           <div class="additional-info-content usa-alert-text">
             {{ entity.fieldBody.processed }}
 
-            {% if entity.fieldAlertOperatingStatusCta == true and statusUrl != false %}
+            {% if entity.fieldAlertOperatingStatusCta == true and statusUrl.length %}
```

Comment:
That's good.  I think "is not empty"  is the more readable test,but this maps exactly to length according to this.  
https://straightupcraft.com/articles/testing-if-something-exists-is-defined-length-is-not-null-is-not-empty